### ISSUE TITLE
 I can't use Tailscale Ping.

### ISSUE THREAD

**Issue by zhangpinzheng** [louislam/uptime-kuma#4229]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

I use deabin to build UPtime kuma and have also installed tailscale, but I can't use Tailscale Ping. I want to ask if I am missing any operation?

### 📝 Error Message(s) or Log

Error checking Tailscale ping: Error: No output from Tailscale ping

### 🐻 Uptime-Kuma Version

1.23.10

### 💻 Operating System and Arch

Debian GNU/Linux 12 x86_64

### 🌐 Browser

EDGE

### 🟩 NodeJS Version

v18.18.2

**Comment by zhangpinzheng** [comment]
root@yun:/www/wwwroot/uptime-kuma# tailscale status
100.66.251.86   yun                  zpz15269797747@ linux   -
100.108.95.127  backup3              zpz15269797747@ linux   active; direct 192.168.21.3:41641, tx 5300 rx 5212
100.81.246.113  beatless             zpz15269797747@ windows offline
100.64.147.145  desktop-v7g2tm2      zpz15269797747@ windows -
100.118.24.74   nas                  zpz15269797747@ linux   offline

**Comment by CommanderStorm** [comment]
Closing as a duplicate of #4228
See also #4231

TLDR: either wait for the next release or go back to 1.23.7
